src/bindings/python/tests/test_graph/test_manager.py

```diff
@@ -133,9 +135,6 @@ def test_serialize_pass_mixed_args_kwargs_v2(request, tmp_path):
     assert model.get_parameters() == res_model.get_parameters()
     assert model.get_ordered_ops() == res_model.get_ordered_ops()
 
-    os.remove(xml_path)
-    os.remove(bin_path)
```

Comment:
Please revert all `os.remove` and add `del` solution. This is required as files created in tests should be cleaned up automatically.

Reply:
PS also a comment in at least one test that points to this PR (explain why `del` is present) would be great.